content/en/about-filecoin/managing-assets/index.md

```diff
@@ -9,13 +9,31 @@ weight: 60
 
 ## Wallets
 
+<!--TODO: request out to Johnny to verify what goes in relref for the link in this section-->
+Wallets provide a way to securely store digital assets. `FIL` is a native token, meaning it runs on its own blockchain and is not compatible with all wallets out of the box. [Wrapped FIL]{{< relref "wrapped-filecoin" >}} allows the token to interact with other blockchains if desired.
+
+### Custodial vs non-custodial
```

Comment:
Now that I see the whole article, I think we can remove this section on custodial vs non-custodial. It's extremely well written! But less immediately relevant for novice users, and more expert users can find this information elsewhere.